### PR TITLE
Use cdn endpoint instead of legacy endpoit for JS snippet

### PR DESCRIPTION
**Pull request by philipp985**
Fix Issue # .

## Changes
According to the docs this should be the current endpoint to use
[https://github.com/microsoft/ApplicationInsights-JS#active-cdn-endpoints](https://github.com/microsoft/ApplicationInsights-JS#active-cdn-endpoints)

### Checklist
- [ ] I ran Unit Tests locally.
- [ ] CHANGELOG.md updated with one line description of the fix, and a link to the original issue if available.

For significant contributions please make sure you have completed the following items:

- [ ] Design discussion issue #
- [ ] Changes in public surface reviewed

The PR will trigger build, unit tests, and functional tests automatically. Please follow [these](https://github.com/Microsoft/ApplicationInsights-dotnet/blob/develop/.github/CONTRIBUTING.md) instructions to build and test locally.

### Notes for authors:
- FxCop and other analyzers will fail the build. To see these errors yourself, compile localy using the Release configuration.

### Notes for reviewers:

- We support [comment build triggers](https://docs.microsoft.com/azure/devops/pipelines/repos/github?view=azure-devops&tabs=yaml#comment-triggers)
  - `/AzurePipelines run` will queue all builds
  - `/AzurePipelines run <pipeline-name>` will queue a specific build

### CODE REVIEW

**Comment by TimothyMothra** [comment]
Hello @philipp985 ,
Your change looks good to me.

FYI, my team (.NET) depends on the JS team to maintain this snippet. 
Most people are still out for the holidays. 
As soon as I can find someone from the JS side to sign-off on this I'll get it merged.

File: NETCORE/src/Microsoft.ApplicationInsights.AspNetCore/Resources.resx [hunk 1]
```diff
@@ -122,10 +122,10 @@
...
...dystatechange=function(e,t){"loaded"!==n.readyState&amp;&amp;"complete"!==n.readyState||i(0,t)},n}();y.ld&lt;0?l.getElementsByTagName("head")[0].appendChild(e):setTimeout(function(){l.getElementsByTagName(u)[0].parentNode.appendChild(e)},y.ld||0)}try{m.cookie=l.cookie}catch(p){}function t(e){for(;e.length;)!function(t){m[t]=function(){var e=arguments;g||m.queue.push(function(){m[t].apply(m,e)})}}(e.pop())}var n="track",r="TrackPage",o="TrackEvent";t([n+"Event",n+"PageView",n+"Exception",n+"Trace",n+"DependencyData",n+"Metric",n+"PageViewPerformance","start"+r,"stop"+r,"start"+o,"stop"+o,"addTelemetryInitializer","setAuthenticatedUserContext","clearAuthenticatedUserContext","flush"]),m.SeverityLevel={Verbose:0,Information:1,Warning:2,Error:3,Critical:4};var s=(d.extensionConfig||{}).ApplicationInsightsAnalytics||{};if(!0!==d[C]&amp;&amp;!0!==s[C]){method="onerror",t(["_"+method]);var c=T[method];T[method]=function(e,t,n,a,i){var r=c&amp;&amp;c(e,t,n,a,i);return!0!==r&amp;&amp;m["_"+method]({message:e,url:t,lineNumber:n,columnNumber:a,error:i}),r},d.autoExceptionInstrumented=!0}return m}(y.cfg);(T[t]=n).queue&amp;&amp;0===n.queue.length&amp;&amp;n.trackPageView({})}(window,document,{
```
Comment (MSNev) [inline NETCORE/src/Microsoft.ApplicationInsights.AspNetCore/Resources.resx:124]: Can you also grab the latest version of the snippet.
This is using version 4 (which has a race condition that can cause JS exceptions) -- see the ```sv: 4``` within the snippet -- the current version is v5.
Current version is located here in the readme or the raw .js is also listed.
https://github.com/microsoft/ApplicationInsights-JS/tree/master/AISKU/snippet

Comment (TimothyMothra) [reply NETCORE/src/Microsoft.ApplicationInsights.AspNetCore/Resources.resx:124]: I wouldn't expect an external contributor to replace the entire snippet. 
I should be able to get to this sometime this week.

**Comment by TimothyMothra** [comment]
@philipp985 thank you for your contribution.
I'm closing this PR in favor of updating the entire snippet (#2518)

**Pull request closed without merging**